git-cliff-core/Cargo.toml

```diff
@@ -55,7 +55,7 @@ serde_json = "1.0.137"
 bincode = "2.0.0-rc.3"
 serde_regex = "1.1.0"
 tera = "1.20.0"
-indexmap = { version = "2.7.1", optional = true }
+indexmap = { version = "2.7.1", features = ["serde"] }
```

Comment:
this isn't optional anymore since `Release` uses it for submodule commits. maybe use hashmap instead? my reasoning was that its used in `Repository` as well, so keep it consistent

Reply:
If we use `hashmap` then the items are not ordered which is probably not something that we want.

Also why enable the `serde` feature?

Reply:
serde is needed now because `Release`  needs to be serializable and as a consequence `indexmap` .